docs-ref-conceptual/intellij/azure-toolkit-for-intellij-installation.md

```diff
@@ -30,7 +30,30 @@ The Azure Toolkit for IntelliJ provides templates and functionality that allow y
 
 There are two methods of installing the Azure Toolkit for IntelliJ: by using the **Settings** dialog box, and by using the **Configure** menu on the start screen. Both installation methods will be demonstrated in the following steps.
 
-[!INCLUDE [azure-toolkit-for-IntelliJ-prerequisites](../includes/azure-toolkit-for-intellij-prerequisites.md)]
+## Prerequisites
+
+The Azure Toolkit for IntelliJ requires to the following software components:
+
+* An [Azure supported Java Development Kit (JDK)](https://aka.ms/azure-jdks)
```

Comment:
The Azul Zulu JDK shouldn't be a prerequisite for IntelliJ. Developers can use any JDK for development purpose.

Reply:
replaced with openjdk and oracle links.